src/index.ts

```diff
@@ -131,22 +160,17 @@ const iso = <S extends AnyNewtype>(): Iso<S, Carrier<S>> =>
 `;
 
 export function getModels(models: Array<Model>, options: GetModelsOptions, prelude: string = ''): string {
-  const declarations = getDeclarations(models, options.isReadonly, options.optionalType || gen.undefinedType);
-  const newtypeDeclarations: NewtypeRawDeclaration[] = declarations.filter(
-    (d): d is NewtypeRawDeclaration => d.kind === 'newtype'
-  );
-  const typeDeclarations: gen.TypeDeclaration[] = declarations.filter(
-    (d): d is gen.TypeDeclaration => d.kind !== 'newtype'
-  );
-  const sortedTypeDeclarations = gen.sort(sortBy(typeDeclarations, ({ name }) => name));
+  _models = models;
+  const declarations = getDeclarations(models, options.isReadonly);
+  const hasNewtypeDeclarations = models.some(m => 'isValueClass' in m && m.isValueClass);
```

Comment:
could reuse `isNewtype` above?

Reply:
Not really, they work on different inputs (I mean, they could both use a separate method, but it's a simple check)